src/Nokia_LCD.cpp

```diff
@@ -47,7 +54,8 @@ Nokia_LCD::Nokia_LCD(const uint8_t dc_pin, const uint8_t ce_pin,
       kUsingBacklight{false},
       kUsingHardwareSPI{true},
       mX_cursor{0},
-      mY_cursor{0} {}
+      mY_cursor{0},
+      currentFont{&nokiaFont} {}
```

Comment:
Please use the existing naming convention and start member variables with `m`, i.e. `mCurrentFont`

Reply:
Sure! By the way, you usually prefix private members with "k". I'm not an expert of C++, do you have some reference talking about these conventions?

Reply:
Actually it is the **constants** I prefix with `k`. Variables get the `m` prefix. So `currentFont` should become `mCurrentFont` not `mcurrentFont`. :smiley:

Reply:
The `m` prefix for member variables is relatively common in different languages. I don't think I follow any particular style (e.g. the Google C++ Style) for this repository.